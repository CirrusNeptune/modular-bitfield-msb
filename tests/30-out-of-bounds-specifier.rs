use modular_bitfield::prelude::*;

#[bitfield(specifier = true, filled = false)]
#[derive(Debug, PartialEq, Eq, Copy, Clone)]
pub struct Header {
    a: B1,
    b: B128,
}

fn main() {}