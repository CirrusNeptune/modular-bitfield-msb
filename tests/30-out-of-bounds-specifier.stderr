error[E0277]: the trait bound `modular_bitfield::private::checks::False: modular_bitfield::private::checks::SpecifierHasAtMost128Bits` is not satisfied
   --> $DIR/30-out-of-bounds-specifier.rs:4:1
    |
4   | #[derive(Debug, PartialEq, Eq, Copy, Clone)]
    | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ the trait `modular_bitfield::private::checks::SpecifierHasAtMost128Bits` is not implemented for `modular_bitfield::private::checks::False`
    |
   ::: $WORKSPACE/src/private/checks.rs
    |
    |     <Self::CheckType as DispatchTrueFalse>::Out: SpecifierHasAtMost128Bits,
    |                                                  ------------------------- required by this bound in `modular_bitfield::private::checks::CheckSpecifierHasAtMost128Bits`
